doc/topics/style-guide.md

```diff
@@ -147,6 +147,56 @@ Not covered by C++ Core Guidelines.
 
 * Test files should be prefixed with `Test`, e.g. `TestModel` instead of `ModelTests`.
 
+## ✏️ Documentation
+
+Not covered by C++ Core Guidelines.
+
+Our documentation is generated by Doxygen before being published [online](https://cesium.com/learn/cesium-native/ref-doc). Therefore, any public API should be documented with Doxygen-compatible comments that use of the following tags (in the order listed):
+
+- `@brief`: Summarize the purpose of the class, function, etc.
+- `@details`: Use for lengthier explanations, after `@brief`.
```

Comment:
I don't see `@details` being used much in native. Usually it's just any paragraphs after `@brief`. Is it different in Unreal?

Reply:
Hm it's not different in Unreal. I guess I included it because I saw `@details` used in CesiumGltfWriter.

https://github.com/CesiumGS/cesium-native/blob/1780e8769942c61cffd1d0fd6eb452703a7a3486/CesiumGltfWriter/include/CesiumGltfWriter/GltfWriter.h#L76

But that's literally the only instance where we use `@details` 😅  And based on @azrogers 's comment it looks like we prefer to use `@remarks` instead. I'll replace this.